ci/buildAndTest.sh

```diff
@@ -1,3 +1,3 @@
 export PATH=$HOME/.local/bin:$CACHE_DIR/z3/bin:$CACHE_DIR/bin:$PATH
-stack install --install-ghc --test --haddock --no-haddock-deps --fast --stack-yaml stack_linux.yaml --work-dir $CACHE_DIR_REL/.stack-work --stack-root $CACHE_DIR/.stack --allow-different-user
+stack install --install-ghc --haddock --test --fast --stack-yaml stack_linux.yaml --work-dir $CACHE_DIR_REL/.stack-work --stack-root $CACHE_DIR/.stack --allow-different-user
```

Comment:
Shouldn't the haddock generation occur after building and testing TorXakis? Generating documentation takes a considerable amount of time, and we could postpone this after the tests.

Reply:
We can, but then Stack performs another build in order to create haddock docs. This is due to Stack using Cabal for haddock, afaik.